app/assets/stylesheets/_layout.scss

```diff
@@ -0,0 +1,19 @@
+html {
+  box-sizing: border-box;
+}
+
+*,
+*::before,
+*::after {
+  box-sizing: inherit;
+}
+
+figure {
+  margin: 0;
+}
+
+img,
+picture {
```

Comment:
I agree with the comments that this isn't layout. Why not split this off into it's own file? I like the proposed `_media.scss`